src/python/grpcio/grpc/_auth.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, credentials):
 
         # Hack to determine if these are JWT creds and we need to pass
         # additional_claims when getting a token
-        self._is_jwt = 'additional_claims' in inspect.getargspec(
+        self._is_jwt = 'additional_claims' in inspect.getargspec(  # pylint: disable=W1505
```

Comment:
Favor `disable=rule-name` instead of rule number.

Reply:
Done